### DISCUSSION TITLE
GriefPrevention treating Towny commands as spam

### DISCUSSION THREAD

**Discussion by AppeazeTheCheese** [GriefPrevention/GriefPrevention#2138] [Q&A]
On the Minecraft server in question, we have the latest version of Grief Prevention (16.18.1) and Towny (0.99.5). When someone with basic permissions tries to claim land for their town by doing either `/t claim` or `/town claim,` they can only successfully run it once, then subsequent times, Grief Prevention prevents the command from executing, even giving them a message stating that "spammers will be banned" after a while.  Is there a way to disable this? My theory is that it's happening because Grief Prevention also has a command with the alias `/t`, but I don't know for sure. Any way I can disable this?

**Comment by Jikoo** [comment]
GP only considers configured slash commands to be spam. Remove `/t` and `/town` from `GriefPrevention.Spam.ChatSlashCommands` if you have added them. By default the list only contains a few commands known to cause chat messages.

**Reply by AppeazeTheCheese** [reply]
Before posting, I looked at the config and saw this section. It's currently set to `/me;/global;/local`; however, it's still flagging `/t` and `/town` as spam for some reason (specifically `/t claim` and `/town claim`), even after multiple server restarts and updating GriefPrevention to the latest version. And before you ask, I am 100% confident that GriefPrevention is the culprit because if I disable it via PlugMan the problem goes away (and the message I mentioned earlier is a default one from GriefPrevention).

**Reply by Jikoo** [reply]
Sounds like a bug then. Could you please open an issue instead?

**Reply by AppeazeTheCheese** [reply]
Sure thing.
